docs/src/adoc/index.adoc

```diff
@@ -7440,6 +7443,71 @@ Global defined attributes may be configured on the factory bean:
 </bean>
 ----
 
+==== Annotation-based configuration
+For annotation based configurations you can add the following method in a
+`@Configuration` class:
+
+[source,java,indent=0]
+----
+@Configuration
+public class JdbiConfiguration {
+    @Bean
+    public Jdbi jdbi(DataSource ds) {
+        ConnectionFactory cf = new SpringConnectionFactory(dataSource); // <1>
+        final Jdbi jdbi = Jdbi.create(cf);
+        /* additional configuration goes here */
+        return jdbi;
+    }
+}
+----
+<1> This connection factory will enable <<jta, JTA support>>.
```

Comment:
I read this as an anchor to the general JTA documentation, not spring-jta. Could we make this `spring-jta` ?